### DISCUSSION TITLE
[Enhancement] [tour] 【Tour】支持自定义上一步下一步的文案

### DISCUSSION THREAD

**Discussion by minchangyong** [element-plus/element-plus#17985] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `tour`

## Description

希望tour的next和previous支持自定义文字。

### Additional comments

_(empty)_

**Comment by warmthsea** [comment]
@minchangyong 

You can custom it.

https://github.com/element-plus/element-plus/pull/17496/files#diff-5ac3e68d4785d7645cd3e748941017f0670dc5f937c38155863202c89c00207dR133

**Comment by Fuphoenixes** [comment]
<img width="1473" alt="image" src="https://github.com/user-attachments/assets/456b8fc6-b512-4698-8923-9914a6daaa09">
传这个chiidren就可以了
<img width="685" alt="image" src="https://github.com/user-attachments/assets/e10da068-64e3-418c-a285-abe4972d7fa0">
这是源码的处理方式
